app/views/user_mailer/account_reset_request.html.erb

```diff
@@ -1,9 +1,9 @@
 <p class="lead">
-  <%= t('user_mailer.account_reset_request.intro_html', app_name: link_to(APP_NAME, IdentityConfig.store.mailer_domain_name, class: 'gray')) %>
+  <%= t('user_mailer.account_reset_request.intro_html', app_name: link_to(APP_NAME, IdentityConfig.store.mailer_domain_name, class: 'gray'), hours: @deactivation_period_hours) %>
 </p>
 
 <p class="lead">
-  <%= t('user_mailer.account_reset_request.cancel', app_name: APP_NAME, confirmation_period: @confirmation_period) %>
+  <%= t('user_mailer.account_reset_request.cancel', app_name: APP_NAME, confirmation_period: @confirmation_period, hours: @deactivation_period_hours) %>
```

Comment:
Am I correct in seeing that `'user_mailer.account_reset_request.cancel'` string does not include `%{hours}` to be interpolated? (nor `confirmation_period` for that matter)

Reply:
Ah, you're right, there is no additional interpolationg going on there. Will remove.